provisioning/roles/kuma/files/vagrant/settings_local.py

```diff
@@ -3,6 +3,8 @@
 
 INTERNAL_IPS = ('127.0.0.1', '192.168.10.1',)
 
+ALLOWED_HOSTS = '*'
+
```

Comment:
Yesterday I was trying to run with `DEBUG = False` to re-create production behavior and I hit this.